### PR TITLE
Added tobias to zenodo

### PR DESCRIPTION
**Pull request by TobiasKadelka**

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -25,6 +25,11 @@
       "name": "Heckner, Marisa",
       "orcid": "TODO"
     }
```
Comment (adswa) [inline .zenodo.json]:
```suggestion
    },
```

Comment (adswa) [reply .zenodo.json]: could you please add this comma? Else, its not valid json :)

**Comment by TobiasKadelka** [comment]
Yes, I did it

**Comment by adswa** [comment]
awesome, thx!

**Pull request merged**